src/__tests__/terminalManagement.spec.ts

```diff
@@ -0,0 +1,155 @@
+import nock from "nock";
+import Client from "../client";
+import { createClient } from "../__mocks__/base";
+import TerminalManagement from "../services/terminalManagement";
+import { AssignTerminalsRequest } from "../typings/terminalManagement/assignTerminalsRequest";
```

Comment:
consider importing the models from the model.ts file rather than including all individual model files.